### DISCUSSION TITLE
How to completely uninstall/ or maybe full reset Consulo installation on Ubuntu?

### DISCUSSION THREAD

**Discussion by feliperb98** [consulo/consulo#457] [Q&A] [answered]
Hello, I did something wrong on my Consulo editing, I have tried to delete and download again to try again, but I think I am not doing the right way, because every time my bad config returns again.

Thank You for your time :)

**Comment by VISTALL** [comment, marked as answer]
Hello. You need drop system directories. Read about them here - https://consulo.dev/basics/boot/boot.directories/

**Reply by feliperb98** [reply]
thank you friend
